src/app/login/login.component.html

```diff
@@ -1,12 +1,20 @@
 <div class="login-layout">
-    <div class="p-field">
-        <input id="emailadress" type="text" pInputText placeholder="{{ 'EMAILADDRESS' | translate }}"> 
-    </div>
-    <div class="p-field">
-        <input type="password" placeholder="{{ 'PASSWORD' | translate }}" pPassword [feedback]=false> 
-    </div>
+    <form [formGroup] = "loginFormGroup">
+        <div class="p-field">
+            <input id="emailadress" type="text" pInputText placeholder="{{ 'EMAILADDRESS' | translate }}" formControlName="email">
+        </div>
+        <div class="p-field">
+            <input type="password" placeholder="{{ 'PASSWORD' | translate }}" pPassword [feedback]=false formControlName="password"> 
+        </div>
 
-    <button pButton type="button" label="{{ 'LOGIN' | translate }}" ></button>
+        <button 
+            pButton 
+            type="button" 
```

Comment:
Wenn du den type rausnimmst, hat der Button automatisch den type "submit". Dann kannst du den Click Handler weglassen und stattdessen oben in <form> ein ngSubmit einfügen: `(ngSubmit)="submit()"`